department-of-veterans-affairs/abd-vro
# MCP-2176: Conditional BIP Api

## What was the problem?
MockBipApi was enabled in all environments

Associated tickets or Slack threads:

- [MCP-2176](https://amida.atlassian.net/browse/MCP-2176)

## How does this fix it?[^1]
MockBipApi is conditionally enabled for test environments. In all other environments, the actual BIP API is accessed.

## How to test this PR
- NA


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
